src/Test/Lint.hs

```diff
@@ -117,8 +117,7 @@ test build obj = do
     crash ["--clean","listing","existance"] ["changed since being depended upon"]
     crash ["needed1"] ["'needed' file required rebuilding"]
     build ["needed2"]
-
-    when False $ whenM hasTracker $ do
+    whenM hasTracker $ do
```

Comment:
Awesome! I've had a TODO: get this working in my local repo for ages, so a great result.